core/src/test/java/org/radargun/smoke/stages/DefineStageTest.java

```diff
@@ -0,0 +1,37 @@
+package org.radargun.smoke.stages;
+
+import org.radargun.DistStageAck;
+import org.radargun.StageResult;
+import org.radargun.stages.ClusteredStageRunner;
+import org.radargun.stages.DefineStage;
+import org.radargun.state.SlaveState;
+import org.radargun.utils.Utils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Matej Cimbora
+ */
+@Test(timeOut = 30000)
+public class DefineStageTest {
+
+   public void smokeTest() throws Exception {
+      ClusteredStageRunner stageRunner = new ClusteredStageRunner(1);
+      SlaveState slaveState = stageRunner.getSlaveState();
+
+      Assert.assertNull(slaveState.get("test"));
+
+      DefineStage defineStage = new DefineStage();
+      Utils.setField(DefineStage.class, "var", defineStage, "test");
```

Comment:
This is not very nice... Rather make that field package-protected and set it directly.